libsplinter/src/admin/rest_api/error.rs

```diff
@@ -18,7 +18,9 @@ use crate::circuit::store;
 
 #[derive(Debug)]
 pub enum ProposalRouteError {
+    #[allow(dead_code)]
```

Comment:
Can these be protected by the conditionals that require them, instead of marking them dead_code? The problem with dead_code is that it gives no clues as to when we should remove these allow statements.

Reply:
updated